app/src/main-process/app-window.ts

```diff
@@ -204,7 +208,11 @@ export class AppWindow {
 
   /** Show the window. */
   public show() {
-    this.window.show()
+    if (this.shouldMaximizeOnShow) {
+      this.window.maximize() // maximize will also call show()
+    } else {
+      this.window.show()
+    }
```

Comment:
Looking at the documentation for `maximize` it says 

> Maximizes the window. This will also show (**but not focus**) the window if it isn't being displayed already.

And the docs for show says

> Shows **and gives focus** to the window.

Which makes me think we should always show and sometimes maximize the window. What do you think about something like this?

```suggestion
    this.window.show()
    if (this.shouldMaximizeOnShow) {
      this.window.maximize()
    }
```

Reply:
Oh, you're right it does not actually call show() but "shows" the window. And you fix looks great, I will commit it right now